content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
+| [DogStatsD][2]    | `dog.count(...)`                     | COUNT           | RATE                |
+| [DogStatsD][2]    | `dog.increment(...)`                 | COUNT           | RATE                |
+| [DogStatsD][2]    | `dog.decrement(...)`                 | COUNT           | RATE                |
+| [Agent check][3]  | `self.count(...)`                    | COUNT           | COUNT               |
+| [Agent check][4]  | `self.monotonic_count(...)`          | COUNT           | COUNT               |
 
-Discover how to submit set metrics:
+**Note**: When submitting a COUNT metric type through DogStatsD, the metric appears as a RATE in-app to ensure relevant comparison across different Agents. Consequently, StatsD counts may appear with a decimal value within Datadog (since they are normalized over a time interval to report units per second).
 
-* [With a custom Agent Check][1]
-* [With DogStatsD][2]
 
-[1]: /developers/metrics/agent_metrics_submission/?tab=set
-[2]: /developers/metrics/dogstatsd_metrics_submission/#set
+[1]: /api/?lang=python#post-timeseries-points
+[2]: /developers/metrics/dogstatsd_metrics_submission/#count
+[3]: /developers/metrics/agent_metrics_submission/?tab=count#count
+[4]: /developers/metrics/agent_metrics_submission/?tab=count#monotonic-count
 {{% /tab %}}
-{{% tab "TIMER" %}}
+{{% tab "RATE" %}}
+
+Submit your RATE type metrics from one of the following sources:
+
+| Submission Source | Submission Method (python)          | Submission Type | Datadog In-App Type |
+| ----------------- | ----------------------------------- | --------------- | ------------------- |
+| [API][1]          | `api.Metric.send(type="rate", ...)` | RATE            | RATE                |
+| [Agent check][2]  | `self.rate(...)`                    | RATE            | GAUGE               |
 
-**The `TIMER` metric type is an implementation of the `HISTOGRAM` metric type within DogStatsD** (not to be confused with timers in the standard StatsD). It measures timing data only: for example, the amount of time a section of code takes to execute, or how long it takes to fully render a page. See the [TIMER DogStatsD documentation][1] to learn how to instrument your code to submit `TIMER`s.
+**Note**: When submitting a RATE metric type through DogStatsD, the metric appears as a GAUGE in-app to ensure relevant comparison across different Agents.
 
-[1]: /developers/metrics/dogstatsd_metrics_submission/#timers
+
+[1]: /api/?lang=python#post-timeseries-points
+[2]: /developers/metrics/agent_metrics_submission/?tab=rate
 {{% /tab %}}
-{{< /tabs >}}
+{{% tab "GAUGE" %}}
 
-## Submission types and Datadog in-app types
+Submit your GAUGE type metrics from one of the following sources:
 
-Datadog accepts metrics submitted from a variety of sources:
+| Submission Source | Submission Method (python)           | Submission Type | Datadog In-App Type |
+| ----------------- | ------------------------------------ | --------------- | ------------------- |
```

Comment:
Alphabetize this table: 

| [Agent check][3]  | `self.gauge(...)`                    | GAUGE           | GAUGE               |
| [API][1]          | `api.Metric.send(type="gauge", ...)` | GAUGE           | GAUGE               |
| [DogStatsD][2]    | `dog.gauge(...)`                     | GAUGE           | GAUGE               |